src/WebJobs.Extensions.DurableTask/DefaultPlatformInformation.cs

```diff
@@ -49,6 +44,44 @@ public bool InLinuxAppService()
             return inLinuxDedicated;
         }
 
+        public OperatingSystem GetOperatingSystem()
+        {
+            if (this.InLinuxConsumption() || this.InLinuxAppService())
+            {
+                return OperatingSystem.Linux;
+            }
+
+            return OperatingSystem.Windows;
+        }
+
+        public bool IsInConsumptionPlan()
+        {
+            return this.InLinuxConsumption() || this.InWindowsConsumption();
+        }
+
+        public WorkerRuntimeType GetWorkerRuntimeType()
+        {
+            string workerRuntime = this.nameResolver.Resolve("FUNCTIONS_WORKER_RUNTIME");
+            if (string.Compare(workerRuntime, "python", ignoreCase: true) == 0)
```

Comment:
nit: it is more common to see `string.Equals(workerRuntime, "python", StringComparison.OrdinalIgnoreCase)`, but this seems to work too.

Reply:
This entire if-else block could probably be represented with [Enum.TryParse](https://docs.microsoft.com/en-us/dotnet/api/system.enum.tryparse?view=net-5.0)